itest/lnd_misc_test.go

```diff
@@ -272,11 +272,22 @@ func testListChannels(ht *lntest.HarnessTest) {
 	// Check the returned response is correct.
 	aliceChannel := ht.QueryChannelByChanPoint(alice, chanPoint)
 
+	// Query the channel again, this time with peer alias lookup.
+	opts := []lntest.ListChannelOption{lntest.WithPeerAliasLookup()}
+
+	aliceChannelWithAlias := ht.QueryChannelByChanPoint(
+		alice, chanPoint, opts...,
```

Comment:
The nice thing about these options is, you can just do `alice, chanPoint, lntest.WithPeerAliasLookup()` here. But non-blocking, since the diff is now much cleaner already.

Reply:
Changed it here and below. I thought having the `opts` field might come in in the future to add options to it. But it isn't very likely that we add additional options right here.